src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -1320,12 +1318,19 @@ internal MethodInfo OnDeserialized
                 }
             }
 
-#if !NET_NATIVE
+
             internal override DataContractDictionary KnownDataContracts
             {
                 [SecurityCritical]
                 get
                 {
+#if NET_NATIVE
```

Comment:
Thanks for pointing this out. I tried to keep the code to behavior as before the fix as much as possible when making the fix. But now I don't think we actually need this #if. I will remove it.